src/components/PatchReviewVote/PatchReviewVote.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import { PatchReviewVotes } from "../../queries/queries";
+
+export default function PatchReviewVote({review_id, votes}) {
+    const [vote, setVote] = useState(votes);
+
+    const HandleUpvote = (e) => {
+        e.preventDefault();
+        if (!review_id) {
+            return null;
+        }
+        setVote((vote) => vote + 1);
+        PatchReviewVotes(review_id, 1);
+    }
+
+    const HandleDownvote = (e) => {
+      e.preventDefault();
+      if (!review_id) {
+        console.log("You shouldn't be seeing this, something has gone horribly wrong!");
```

Comment:
We will need to handle errors slightly differently, errors should not be logged to the console.

Reply:
Removed this as this shouldn't be possible, was only there for while I was trying to fix it.